exercises/concept/treasure-chest/src/TreasureChest.elm

```diff
@@ -0,0 +1,28 @@
+module TreasureChest exposing (..)
+
+{-|
+
+1.  define a single variant generic TreasureChest custom type with associated data:
+      - a String (for a password)
+
+      - a generic (for the treasure)
+
+-}
+
+
+{-|
+
+1.  define getTreasure
```

Comment:
This should be 2.
But really, having this comment and a large `Debug.todo` looks weird, so maybe we could either drop one, or maybe swap

```elm
{-|
2.  define getTreasure
Define getTreasure, with a type annotation, with two parameters:
- a String (for the passwordAttempt)
- a TreasureChest custom type
 and returns:
- if the passwordAttempt does not match the String in the TreasureChest
  - Nothing
- if the passwordAttempt matches the String in the TreasureChest
  - Just treasure (where treasure is the generic in the TreasureChest)
-}

getTreasure passwordAttempt =
  Debug.todo "Please implement the getTreasure function"

```

Reply:
> This should be 2.

Interestingly this auto corrects to 1, so I've used words instead, and changed the comments as you suggest.